README.rst

```diff
@@ -38,7 +38,7 @@ Install ``pulp-gem`` from source
 
 1)  sudo -u pulp -i
 2)  source ~/pulpvenv/bin/activate
-3)  git clone https://github.com/pulp/pulp\_gem.git
+3)  git clone https://github.com/ATIX-AG/pulp_gem
```

Comment:
Thanks. But i think, the backslash is still needed for the reST parser.
Mind to add yourself to the AUTHORS?